application/controllers/admin/emailtemplates.php

```diff
@@ -63,10 +63,19 @@ function index($iSurveyId)
         } else {
             $sEscapeMode = 'unescaped';
         }
+        $uploadDir = realpath(Yii::app()->getConfig('uploaddir'));
         foreach ($grplangs as $key => $grouplang) {
             $aData['bplangs'][$key] = $grouplang;
             $aData['attrib'][$key] = SurveyLanguageSetting::model()->find('surveyls_survey_id = :ssid AND surveyls_language = :ls', array(':ssid' => $iSurveyId, ':ls' => $grouplang));
-            $aData['attrib'][$key]['attachments'] = unserialize($aData['attrib'][$key]['attachments']);
+            $attachments = unserialize($aData['attrib'][$key]['attachments']);
+            foreach ($attachments as &$template) {
+                foreach ($template as &$attachment) {
```

Comment:
PHP Error [500]

Invalid argument supplied for foreach() (C:\xampp\htdocs\limesurvey400\application\controllers\admin\emailtemplates.php:71)

#0 C:\xampp\htdocs\limesurvey400\framework\base\CApplication.php(834): CErrorHandler->handle()
#1 C:\xampp\htdocs\limesurvey400\application\controllers\admin\emailtemplates.php(71): LSYii_Application->handleError()
#2 unknown(0): emailtemplates->index()
#3 C:\xampp\htdocs\limesurvey400\framework\web\actions\CAction.php(109): ReflectionMethod->invokeArgs()
#4 C:\xampp\htdocs\limesurvey400\application\core\Survey_Common_Action.php(86): emailtemplates->runWithParamsInternal()
#5 C:\xampp\htdocs\limesurvey400\framework\web\CController.php(308): emailtemplates->runWithParams()
#6 C:\xampp\htdocs\limesurvey400\framework\web\CController.php(286): AdminController->runAction()
#7 C:\xampp\htdocs\limesurvey400\framework\web\CController.php(265): AdminController->runActionWithFilters()
#8 C:\xampp\htdocs\limesurvey400\application\controllers\AdminController.php(202): AdminController->run()
#9 C:\xampp\htdocs\limesurvey400\framework\web\CWebApplication.php(282): AdminController->run()
#10 C:\xampp\htdocs\limesurvey400\framework\web\CWebApplication.php(141): LSYii_Application->runController()
#11 C:\xampp\htdocs\limesurvey400\framework\base\CApplication.php(185): LSYii_Application->processRequest()
#12 C:\xampp\htdocs\limesurvey400\index.php(182): LSYii_Application->run()

Reply:
Should be fixed now